modules/installation-downloading-image-restricted.adoc

```diff
@@ -22,9 +22,10 @@ environment.
 .Prerequisites
 
 * Obtain the {product-title} installation program. For a restricted network installation, the program is on your mirror registry host.
+* You have access to an HTTP server that you can access from your computer.
 
 .Procedure
-
+ifndef::vsphere[]
```

Comment:
Added ifndef statement in the event that these steps are not needed in a diconnected environment where a customer might be offline.